src/clouds.cpp

```diff
@@ -361,6 +361,22 @@ void Clouds::update(v2f camera_p, video::SColorf color_diffuse)
 	m_color.b = MYMIN(MYMAX(color_diffuse.b * m_params.color_bright.getBlue(),
 			m_params.color_ambient.getBlue()), 255) / 255.0f;
 	m_color.a = m_params.color_bright.getAlpha() / 255.0f;
+
+	// is the camera inside the cloud mesh?
+	m_camera_inside_cloud = false; // default
+	if (m_enable_3d)
+	{
```

Comment:
Yes, sorry, I did mention it was still in the early churning phase. I use vim, so getting clang-format for only a few lines is often tricky, which is why I save it for later.